src/main/java/org/cryptomator/frontend/fuse/FileAttributesUtil.java

```diff
@@ -78,4 +79,36 @@ public void copyBasicFileAttributesFromNioToFuse(BasicFileAttributes attrs, File
 		}
 	}
 
+	public long posixPermissionsToMode(Set<PosixFilePermission> permissions) {
```

Comment:
I would call the method `posixPermissionsToOctalMode` to keep an coherent naming scheme